certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -4784,83 +5013,183 @@
         "description": "Get the operation status of for the operation ID provided."
       }
     },
-    "/api/v1/Trigger/ManageWebHook": {
-      "post": {
+    "/api/v1/Mgmt/GetOperationStatusGetTrackedChanges": {
+      "get": {
         "tags": [
-          "Trigger"
+          "Mgmt"
         ],
-        "operationId": "ManageWebHookAsync",
+        "operationId": "GetOperationStatusGetTrackedChanges",
         "consumes": [
           "application/json"
         ],
         "produces": [
           "application/json"
         ],
         "parameters": [
+          {
+            "in": "query",
+            "name": "operationId",
+            "description": "Retrieve the operation status of a GetOperationStatusGetTrackedChanges operation for the Operation ID provided.",
+            "type": "string"
+          },
           {
             "in": "body",
-            "name": "request",
-            "description": "",
-            "schema": {
-              "$ref": "#/definitions/DtoManageWebHook"
-            }
+            "name": "body",
+            "schema": { }
```

Comment:
@Encodian can you please explain why you have an empty schema? please define a proper schema.

Reply:
@Amjed-Ayoub - It is empty because there is no value passed in the body. This is exactly the same as the previously approved '/api/v1/Mgmt/GetOperationStatus***' actions, there is around 20 of them. I would request this is also approved as it matches previously approved convention.